contrib/kserve/kserve/kserve_kubeflow.yaml

```diff
@@ -20110,7 +20630,8 @@ apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   annotations:
-    cert-manager.io/inject-ca-from: kubeflow/serving-cert
+    cert-manager.io/inject-ca-from: kserve/serving-cert
```

Comment:
There are several places where annotation is `cert-manager.io/inject-ca-from: kserve/serving-cert`, but should be `cert-manager.io/inject-ca-from: kubeflow/serving-cert`

Reply:
Resolving, see my [other comment](https://github.com/kubeflow/manifests/pull/2504#discussion_r1299977451)